File: bounded-contexts/node/src/entities/cart-Item.ts
import { Quantity } from './quantity'
import { ItemId } from './item-id'

export class CartItem {
  readonly #itemId: ItemId
  readonly #quantity: Quantity

  constructor(itemId: ItemId, quantity: Quantity) {
    this.#itemId = itemId
    this.#quantity = quantity
  }

  get quantity() {
    return this.#quantity
  }

  get itemId(): ItemId {
    return this.#itemId
  }
}

File: bounded-contexts/dotnet/MyShop.Services/Shipping/CartItem.cs
using MyShop.Entities;

namespace MyShop.Services.Shipping
{
    public class CartItem
    {
        public CartItem(ItemId itemId, int quantity)
        {
            ItemId = itemId;
            Quantity = quantity;
        }

        public ItemId ItemId { get; }
        public int Quantity { get; }
    }
}
